using-mobile-extensions/adobe-experience-platform-assurance/README.md

```diff
@@ -167,35 +247,105 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 {% endtab %}
 
 {% tab title="React Native" %}
-{% hint style="danger" %}
-Updated support for React Native is coming. This section will be updated shortly.
-{% endhint %}
+
+### JavaScript
+
+When using React Native, registering AEPAssurance with Mobile Core should be done in native code which is shown under the Android and iOS tabs.
```

Comment:
Change to:

When using React Native, register AEP Assurance with Mobile Core in native code as shown on the Android and iOS tabs. (and add a link).

Make same change for other platforms below.